MatrixSDKTests/MXCryptoTests.m

```diff
@@ -3007,6 +3009,52 @@ - (void)testDiscardAndRestoreOlmOutboundKey
     }];
 }
 
+#pragma mark - One time / fallback keys
+
+- (void)testFallbackKeySignatures
+{
+    [MXSDKOptions sharedInstance].enableCryptoWhenStartingMXSession = YES;
```

Comment:
We need to reset it to `NO` after the MXSession creation to avoid to affect other tests. Yeah, singleton...

Reply:
Ha, good one 😅